lib/code_corps_web/controllers/stripe_connect_plan_controller.ex

```diff
@@ -1,21 +1,29 @@
 defmodule CodeCorpsWeb.StripeConnectPlanController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  alias CodeCorps.StripeConnectPlan
+  alias CodeCorps.{StripeConnectPlan, User}
   alias CodeCorps.StripeService.StripeConnectPlanService
 
-  plug :load_and_authorize_changeset, model: StripeConnectPlan, only: [:create]
-  plug :load_and_authorize_resource, model: StripeConnectPlan, only: [:show]
-  plug JaResource
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
+  plug CodeCorpsWeb.Plug.IdsToIntegers
 
-  @spec model :: module
-  def model, do: CodeCorps.StripeConnectPlan
+  @spec show(Conn.t, map) :: Conn.t
+  def show(%Conn{} = conn, %{"id" => id} = params) do
+    with %User{} = current_user <- conn |> Guardian.Plug.current_resource,
+         %StripeConnectPlan{} = stripe_platform_plan <- StripeConnectPlan |> Repo.get(id),
+         {:ok, :authorized} <- current_user |> Policy.authorize(:show, stripe_platform_plan, params) do
```

Comment:
You don't need to pass in the params here, since they aren't used and `authorize?` already has a default.